multiapps-controller-persistence/src/main/resources/org/cloudfoundry/multiapps/controller/persistence/db/changelog/db-changelog-1.157.0-persistence.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+         http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.6.xsd">
+
+    <changeSet author="sap.com" id="add_table_async_upload_jobs">
+        <preConditions onFail="MARK_RAN">
+            <not>
+                <tableExists tableName="async_upload_jobs" />
+            </not>
+        </preConditions>
+        <createTable tableName="async_upload_jobs">
+            <column name="id" type="VARCHAR(255)">
+                <constraints primaryKey="true" primaryKeyName="async_upload_jobs_pk" />
+            </column>
+            <column name="state" type="VARCHAR(64)">
+                <constraints nullable="false" />
+            </column>
+            <column name="user" type="VARCHAR(255)">
+                <constraints nullable="false" />
+            </column>
+            <column name="started_at" type="TIMESTAMP">
+                <constraints nullable="false" />
+            </column>
+            <column name="space_guid" type="VARCHAR(255)">
+                <constraints nullable="false" />
+            </column>
+            <column name="finished_at" type="TIMESTAMP" />
+            <column name="namespace" type="TEXT" />
+            <column name="mta_id" type="TEXT" />
```

Comment:
I know users are creative with their names but in the other tables mta_id and namspace are varchar. I do not think TEXT is needed